### PR TITLE
feat: access watchers info directly

### LINKED ISSUE #35: Endpoint for getting watchers
`/watchers`

Should just return an array of objects 

```js
[
  {
    name: String,
    displayName: String,
    description: String
  },
]`

### LINKED ISSUE #48: Move watchers download list to config

### LINKED ISSUE #63: Expose watchers to whole application in easier way
A file should be in charge of keeping track of all watchers and exposing them in easy to use ways like

`watchers.list` -> an array with all watchers just name and description

`watchers[watcherName]` -> direct access to one watcher

and things like that.

Also do #48 while doing this

### PR DESCRIPTION
**Pull request by negebauer**
Closes #63 
Closes #48 
Closes #35

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -27,6 +27,33 @@ function databaseUrl() {
   return 'mongodb://localhost/notify-watcher';
 }
 
+// We can clone a privaty repo using the following url format
```
Comment (sasalatart) [inline src/config.js]:
```suggestion
// We can clone a private repo using the following url format
```

File: src/config.js [hunk 2]
```diff
@@ -27,6 +27,33 @@ function databaseUrl() {
...
+ * ```js
+ * {
+ *  url: 'The repo url to clone',
+ *  branch: '(Optional) The branch of the repo to checkout',
+ *  commit: '(Optional) The commit of the repo to checkout',
+ *  watchers: 'Array of watchers names or the WATCHERS_LIST_DOWNLOAD_ALL_KEY string'
+ * }
+ * ```
+ *
+ * Either `branch` or `commit` **MUST** be defined, if both are not
```
Comment (sasalatart) [inline src/config.js]: Can't we just assume in that case that the latest commit from the master branch will be used?

File: src/routes/watchers.test.js [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
-      request.get('/watchers').expect(HTTP_CODES.ok));
+    const { watchersList } = loadWatchers();
+    let response;
+
+    beforeAll(async () => {
+      response = await request.get('/watchers');
+    });
+
+    it('should return "ok"', () => expect(response.status).toBe(HTTP_CODES.ok));
+    it('should be an array', () => expect(response.body).toBeInstanceOf(Array));
```
Comment (sasalatart) [inline src/routes/watchers.test.js]: You can also check that every element in the array follows a structure, via something like `expect(array.every).toBeTruthy()`

File: src/watchers/load-watchers.js [hunk 4]
```diff
@@ -40,6 +45,16 @@ function loadWatchers(watchersPath) {
       validateTimeframe(config.timeframe, name, { verbose: true }),
     );
 
+  const watchersList = watchers.map(({ name, displayName, description }) => ({
+    description,
+    displayName,
+    name,
+  }));
+  const watchersObject = watchers.reduce(
+    (object, watcher) => ({ ...object, [watcher.name]: watcher }),
```
Comment (sasalatart) [inline src/watchers/load-watchers.js]: By using `_.pick` here you can save yourself the previous map.

File: src/config.js (hunk 2 above)
Comment (negebauer) [reply src/config.js]: we can, added https://github.com/notify-watcher/server/issues/66 for this

**Pull request merged**